trojsten/top30/tests.py

```diff
@@ -56,14 +56,23 @@ def setUp(self):
         self.group.user_set.add(self.users[3])
         self.users[2].groups.add(self.group)
         self.users[3].groups.add(self.group)
```

Comment:
Netreba tu dať save() na modified users? Tiez je nutne pridavat users do group.user_set a zaroven pridavat groups users?

Reply:
Naozaj neviem. V `reviews` je použité len `.user_set.add`, v `people` len `.groups.add`, v `submits` sú obe.

Reply:
Tak asi pokiaľ to robí to čo má tak je to v pohode.